pkg/svcat/service-catalog/broker.go

```diff
@@ -239,14 +240,14 @@ func (sdk *SDK) Sync(name string, scopeOpts ScopeOptions, retries int) error {
 		}
 
 		if scopeOpts.Scope.Matches(ClusterScope) {
-			broker, err := sdk.RetrieveBroker(name)
+			var broker *v1beta1.ClusterServiceBroker
+			broker, err = sdk.RetrieveBroker(name)
 			if err == nil {
 				broker.Spec.RelistRequests = broker.Spec.RelistRequests + 1
 
 				_, err = sdk.ServiceCatalog().ClusterServiceBrokers().Update(broker)
 				if err == nil {
 					success = true
-					fmt.Printf("synced clusterservicebroker (%s)\n", name)
```

Comment:
Why are we removing this print statement here?

Reply:
@shubheksha I should have left a comment or note about this in the PR. 
I went to correct the text (caught it on a late code review) and then found we were already printing a good message in the calling function:

https://github.com/kubernetes-incubator/service-catalog/blob/f3a7a072ac60d9ccb7bd55b05ada0de18db3cab1/cmd/svcat/broker/sync_cmd.go#L75

I verified before and after removing this line, it was printing both messages.